content/addons.md

```diff
@@ -1,14 +1,142 @@
 # Addons
 
-<!-- What is an addon, link to Ember Observer -->
+The modern web developer has an incredible selection of open source code that they can use through the npm ecosystem, plus great tools for writing their own modules. Ember developers are free to use regular npm packages in their apps, but there are also thousands of packages that are made specifically for the Ember ecosystem. We call such packages "addons."
 
-## Learning how to write an addon
+An addon can include everything from JavaScript code, compiling tools, deployment pipelines, templates, stylesheets, and more.
 
-<!-- Suggest looking at source code of other addons -->
+In this guide, we'll cover:
+- Finding and using community addons
+- Addon file structure
+- Writing your an addon within an app
+- Writing an addon that can be shared
+- Turning a regular npm package into an addon
+- Testing addons
+- Managing assets
+- and more
 
-## Addon file structure
+## Finding and using community addons
 
-<!-- what are different sections actually used for -->
+[Ember Observer](https://www.emberobserver.com/) is the definitive way to explore community addons. Although addons can be found by [searching the npm repository directly](https://www.npmjs.com/search?q=ember), Ember Observer has ranked lists of most popular addons and ratings to help developers choose between them. Most are made to drop right into your app with zero configuration. Many addons are backwards-compatible to earlier versions of Ember too!
+
+To install an addon, use:
+
+```bash
+ember install <addon-name>
+```
+
+To be safe, it's a good idea to restart the local server after installing new dependencies, and especially before trying to debug an addon that isn't working.
+
+The `install` command creates an entry in the app's `package.json`, and downloads the addon and its dependencies into `node_modules`. Some addons also create new files or make modifications when they are installed.
```

Comment:
Every now and then the question pops up - especially by people new to Ember - what the difference between `npm install` and `ember install` is. You are already describing here that the `install` command might be responsible for addon-specific modifications. Do you think it might make sense to use this explanation to also make it clearer that this is something special to `ember install`?

Something similar to the following might work:

```
The `ember install` command is quite similar to the `npm install` instruction you might already be familiar with from installing other packages from the npm registry: An `ember install` also creates an entry in the `package.json`, and downloads the addon and its dependencies into `node_modules`. But it does even more; some addons also create new files or make modifications to your app when they are installed via `ember install`. 
```